buildpacks/jvm/src/main.rs

```diff
@@ -137,19 +131,6 @@ impl Buildpack for OpenJdkBuildpack {
             },
         )?;
 
-        libherokubuildpack::log::log_header("Installing Heroku JVM metrics agent");
-
-        if boolean_buildpack_config_env_var(
-            context.platform.env(),
-            SKIP_HEROKU_JVM_METRICS_AGENT_INSTALLATION_ENV_VAR_NAME,
-        ) {
-            libherokubuildpack::log::log_info(format!(
-                "Skipping agent installation, {SKIP_HEROKU_JVM_METRICS_AGENT_INSTALLATION_ENV_VAR_NAME} environment variable is set to a truthy value."
-            ));
-        } else {
-            context.handle_layer(layer_name!("heroku_metrics_agent"), HerokuMetricsAgentLayer)?;
```

Comment:
I think https://github.com/heroku/buildpacks-jvm/blob/main/buildpacks/jvm/src/bin/heroku_metrics_agent_setup.rs can be removed too?

Reply:
Absolutely! Good catch, thanks!